src/components/dialogs/inputs/reactive-capability-curve-table.js

```diff
@@ -33,6 +33,44 @@ function hasValueCorrectFormat(valueToTest, index) {
     );
 }
 
+function getPreviousValue(displayedPreviousValues, displayedValues, index) {
```

Comment:
Can you explain the logic behind this function ? Why do we get the last array field in the first IF instead the indexed one ?